src/main/java/net/minecraftforge/fml/common/registry/GameRegistry.java

```diff
@@ -191,13 +191,8 @@ public static Block registerBlock(Block block, Class<? extends ItemBlock> itemcl
      */
     public static Block registerBlock(Block block, Class<? extends ItemBlock> itemclass, String name, Object... itemCtorArgs)
     {
-        if (Loader.instance().isInState(LoaderState.CONSTRUCTING))
-        {
-            FMLLog.warning("The mod %s is attempting to register a block whilst it it being constructed. This is bad modding practice - please use a proper mod lifecycle event.", Loader.instance().activeModContainer());
-        }
         try
         {
-            assert block != null : "registerBlock: block cannot be null";
```

Comment:
If a null block is registered it's going to crash anyway, so we might as well be descriptive